kong/metadata.csv

```diff
@@ -1,7 +1,13 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
 kong.bandwidth.count,count,,byte,,[OpenMetrics V2] The total bandwidth in bytes consumed per service/route in Kong,0,kong,,
+kong.bandwidth.bytes.count,count,,byte,,[OpenMetrics V2] (Kong V3) The total bandwidth in bytes consumed per service/route in Kong,0,kong,,
```

Comment:
This is essentially a rename of the metric name right above. Maybe we can indicate in the old metric's description that it only applies to versions < 3? The same applies to other metrics that we know won't be available for Kong v3+.

Also, for new metrics, it might be better to indicate the version as v3+ (assuming that newer versions of kong will also keep these). Like this:

```suggestion
kong.bandwidth.bytes.count,count,,byte,,[OpenMetrics V2] (Kong v3+) The total bandwidth in bytes consumed per service/route in Kong,0,kong,,
```

Reply:
Done, added a `(Kong v2<)` to the older metrics; can change it if we want a different marker.

Reply:
I don't find that notation clear enough; if version 3 is where these metrics disappeared, it might be better to mark it as `(Kong < 3)`.

(And I think this is ready to merge after that)

Reply:
@alopezz Done!